README.md

```diff
@@ -639,6 +639,21 @@ function Header() {
 }
 ```
 
+It is generally a good idea to use the above method for importing assets, because:
+
+- throws a compilation error when an asset is missing.
+- filenames include hashing for browser caching.
+
+If you cannot use this method and want to place some of your assets in the [publicDirectory](#publicdirectory), you can create links with the help of the in built environment variable `BASE_PATH`:
+
+```js
+function Header() {
+  return <img src={process.env.BASE_PATH + '/logo.png'} alt="Logo" />;
+}
+```
+
+The environment variable `BASE_PATH` is automatically set for you and is equivalent to the value of [`siteBasePath`](#sitebasepath).
```

Comment:
We should explain that it will never end with a slash, even if your `siteBasePath` does.